vendor/woocommerce/action-scheduler/classes/WP_CLI/Action/Delete_Command.php

```
@@ -0,0 +1,108 @@
+		add_action( 'action_scheduler_deleted_action', array( $this, 'on_action_deleted' ) );
+	}
+
+	/**
+	 * Execute.
+	 *
+	 * @return void
+	 */
+	public function execute() {
+		$store = \ActionScheduler::store();
+
+		$progress_bar = \WP_CLI\Utils\make_progress_bar(
```

PHPStan (2.1.46)

[error] Function WP_CLI\Utils\make_progress_bar not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)